contrib/libmpdclient/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "libmpdclient"
+pkgver = "2.22"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "ninja", "pkgconf"]
+pkgdesc = "Asynchronous API library for interfacing MPD in the C, C+"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "BSD-2-Clause AND BSD-3-Clause"
+url = "https://musicpd.org/libs/libmpdclient"
+source = (
+    f"https://www.musicpd.org/download/{pkgname}/2/{pkgname}-{pkgver}.tar.xz"
+)
+sha256 = "eac15b82b5ba5ed0648af580221eb74657394f7fe768e966d9e9ebb27435429f"
+# no checks
+options = ["!check"]
```

Comment:
it literally has tests

Reply:
(you need check-devel and -Dtest=true for it. please actually check this, it takes like 20 seconds..)